### ISSUE TITLE
processing additional params in server-side datatables-query

### ISSUE THREAD

**Issue by NavneetKaur9** [fiddus/datatables-query#3]
I'm sending one additional parameter to the server .For eg: i have a collection named "bills" in which i'm storing the data of all my users with a specific "bill_id" and now i want to retrieve all bill details associated with one particular "bill_id".
So , i wanna know how can i get data based on key "bill_id", m already sending this additional param from front-end.

**Comment by vinicius0026** [comment]
@NavneetKaur9 How are you sending this parameter? I'm assuming `bill_id` is unique, so your query would return only one document, is that right? And you want pagination in some array inside this document? Please provide further explanation about your use case and data model, so we can figure some solution.

**Comment by NavneetKaur9** [comment]
suppose this is the data model : 
{"_id":"637789222", name: "john", bill_paid: "$10", bill_for: "mobile", bill_dated:"06/06/2016"}, {"_id":"637789223", name: "julie", bill_paid: "$20", bill_for: "mobile", bill_dated:"06/07/2016"}, {"_id":"637789224", name: "john", bill_paid: "$30", bill_for: "laptop", bill_dated:"06/08/2016"}, {"_id":"637789225", name: "john", bill_paid: "$40", bill_for: "internet", bill_dated:"06/09/2016"}, {"_id":"637789226", name: "julie", bill_paid: "$50", bill_for: "datacard", bill_dated:"06/09/2016"}

Now i want to construct a datatable showing entries for only one name(suppose john),
then i want my table to show only these three entries:-
{"_id":"637789222", name: "john", bill_paid: "$10", bill_for: "mobile", bill_dated:"06/06/2016"},
{"_id":"637789224", name: "john", bill_paid: "$30", bill_for: "laptop", bill_dated:"06/08/2016"}, {"_id":"637789225", name: "john", bill_paid: "$40", bill_for: "internet", bill_dated:"06/09/2016"}

in short, i want to construct different table for diferent users.
So, how can i send this additional "name" parameter to the server end so that i can retrieve data for a particular user.

**Comment by vinicius0026** [comment]
@NavneetKaur9 the way this module is implemented right now, you could achieve what you want by sending a search parameter with the user name. Be sure to set all other fields to not searchable.
In jQuery, it would be something like this:

``` javascript
$('#example').dataTable({
  "columns": [{
    "data": "_id",
    "searchable": "false"
  }, {
    "data": "name",
    "searchable": "true"
  }, {
    "data": "bill_paid",
    "searchable": "false"
  }, {
    "data": "bill_for",
    "searchable": "false"
  }, {
    "data": "bill_dated",
    "searchable": "false"
  }],
  "search": {
    "search": "john" // the name you want to filter
  }
});
```

**Comment by vinicius0026** [comment]
@NavneetKaur9 I'm closing this issue since it has been inactive for quite some time.
